common/main.c

```diff
@@ -5,19 +5,32 @@
 
 #include <compiler.h>
 #include <console.h>
+#include <debug.h>
 #include <dm.h>
 #include <stdbool.h>
 #include <work.h>
+#include <drivers/watchdog.h>
 #include <platform/devices.h>
 
+#define WDOG_TIMEOUT (5 * 1000 * 1000) * 24 /* 5 seconds */
```

Comment:
The guarding parentheses go on the outside of the entire expression.

Reply:
Fixed.